core/templates/dev/head/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -17,72 +17,91 @@
  * contributors to contribute.
  */
 
-require('domain/opportunity/ExplorationOpportunitySummaryObjectFactory.ts');
-require('domain/opportunity/SkillOpportunityObjectFactory.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {
+  UrlInterpolationService
+} from 'domain/utilities/url-interpolation.service';
+import {
+  ExplorationOpportunitySummary
+} from 'domain/opportunity/ExplorationOpportunitySummaryObjectFactory';
+import {
+  SkillOpportunity
```

Comment:
Please check the other files and use that style.
![image](https://user-images.githubusercontent.com/16653571/75092567-01686000-559f-11ea-8fd5-1f1b3c2c434a.png)

Reply:
Done